src/assets/scripts/base/PositionModel.js

```diff
@@ -186,46 +189,17 @@ export default class PositionModel {
         const d_lat = degreesToRadians(lat_a - lat_b);
         const d_lng = degreesToRadians(lng_a - lng_b);
 
-        // TODO: what do these vars mean? a & c?
-        // TODO: could the maths here be abstracted?
+        // TODO: what is actually getting set to `a` here? and what does `a` represent?
```

Comment:
Most likely a simple haversine calculation, where `a` and `b` are the lines from the earth center to two different points, and `c` is the length of the arc between (great circle distance).

[Example JS haversine implementation](http://stackoverflow.com/questions/27928/calculate-distance-between-two-latitude-longitude-points-haversine-formula)